openreview/conference/builder.py

```diff
@@ -2398,51 +2398,53 @@ def __init__(self,
     allow_public_comments=False,
     anonymous=False,
     unsubmitted_reviewers=False,
+    submitted_reviewers=False,
```

Comment:
I think it is time to refactor this a little bit, there are too many params that are too confusing. We could define the two parameters `participants` and `readers`. These two parameters are arrays that contain that elements from the Participants and Readers class, for example here: https://github.com/openreview/openreview-py/blob/master/openreview/conference/builder.py#L1887


You can check the SubmissionStage or ReviewStage classes to see how we handle the reviewers. 

From the participants array we are going to define the invitees and build the regex for signatures. 

From the readers array we are going to define the list of the dropdown. 

the parameters: anonymous, unsubmitted_reviewers, submitted_reviewers, authors, area_chairs, senior_area_chairs, additional_readers should be mapped to these two parameters.